docs/UserGuide.md

```diff
@@ -105,17 +105,25 @@ The `Tool Bar` at the top provides a set of configuration options that control t
   * Multiple keywords/terms can be used, separated by spaces.
   * Entries that contain _any_ (not necessarily _all_) of the search terms will be displayed.
   * The keywords used to filter author and repository are case-insensitive.
-* `Sort by` : sorting criteria for the Chart Panel
-  * `Total Contribution` : the amount of lines, written by the author, in the repository.
-  * `Variance` : the [variance](https://en.wikipedia.org/wiki/Variance) of the number of lines that the author has contributed to the repository daily, sorted from low to high.
-  * `Author Name` : the author's display name.
-  * `Repo/Branch Name` : see note [1] below.
+* `Group by` : denotes whether to group the captured results by
+  * `None` : results will not be grouped in any particular way.
+  * `Repo/Branch` : results will be grouped by repositories and its' associating branches.
+  * `Author` : results will be  grouped by the name of the author. Contributions made to multiple repositories by a particular author will be grouped under the author.  
+* `Sort groups by`: sorting criteria for the main group
+  * `Name` : groups will be sorted by GitHub ID in alphabetical order.
+  * `Repo/branch` : groups will be sorted in alphabetical order by the name of the repo, followed by name of the branch. See note [1] below.
+  * `Contribution` : groups will be sorted by the total combined contributions within a group, in the order of number of lines added
```

Comment:
groups will be sorted by the ~total~ combined contributions of each group, based on the number of lines written by the authors